### PR TITLE
integration: custom_build integration test

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

**Comment by jazzdan** [comment]
Wow this was actually ... pretty easy.

File: integration/oneup_custom_test.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//+build integration
+
+package integration
+
+import "testing"
+
+func TestOneUpCustom(t *testing.T) {
+	f := newFixture(t, "oneup_custom")
```
Comment (nicks) [inline integration/oneup_custom_test.go]: could you also add a CurlUntil like in the other integration tests to ensure the server comes up?

File: integration/oneup_custom/Tiltfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# -*- mode: Python -*-
+
+custom_build(
+  'gcr.io/windmill-test-containers/integration/oneup-custom',
+  'docker build -t $TAG .',
```
Comment (nicks) [inline integration/oneup_custom/Tiltfile]: out of curiosity, how does this image get pushed?

Comment (nicks) [reply integration/oneup_custom/Tiltfile]: (a better test might be to use a non-docker image builder...)

File: integration/oneup_custom_test.go (hunk 1 above)
Comment (jazzdan) [reply integration/oneup_custom_test.go]: Whoops, I added that to the wrong file locally and didn't commit it. 😬

File: integration/fixture_test.go [hunk 3]
```diff
@@ -220,7 +221,10 @@ func (f *fixture) AllPodsReady(ctx context.Context, selector string) (bool, stri
 	cmd := exec.Command("kubectl", "get", "pods",
 		namespaceFlag, "--selector="+selector, "-o=template",
 		"--template", "{{range .items}}{{.metadata.name}} {{.status.phase}}{{println}}{{end}}")
-	out, err := cmd.Output()
+	log.Printf("CMD: %+v", cmd.Args)
+	out, err := cmd.CombinedOutput()
```
Comment (jazzdan) [inline integration/fixture_test.go]: whoops, will remove these

File: integration/oneup_custom/Tiltfile (hunk 2 above)
Comment (jazzdan) [reply integration/oneup_custom/Tiltfile]: So the builder returns a ref that the image_build_and_deployer pushes, just like the regular docker_build. :)

**Pull request merged**